website/versioned_docs/version-7.0.0/cli.md

```diff
@@ -100,15 +100,23 @@ npx babel src --out-file script-compiled.js
 
 ### Ignore files
 
-Ignore spec and test files
+Given files will be ignored from compilation and will not moved to destination folder.
```

Comment:
One tricky thing with this page, is that these are basically "recipes" for doing common things as opposed to a full detailed list of options.

Should we just keep the original text to match and maybe add a more noticeable section above this that links to [options](https://babeljs.io/docs/en/options)?